bookdown/06-special-tasks/06-special-tasks.Rmd

```diff
@@ -1 +1,53 @@
 # Special Tasks {#special-tasks}
+
+This chapter explores the different functions of mlr3 when dealing with specific data sets that require further statistical modification to undertake sensible analysis.
+Following topics are discussed: 
+
+##Survival Analysis 
+
+[Survival analysis](#survivial) is used to monitor the period of time until a specific event takes places.
+This specific event could be e.g. death, transmission of a disease, marriage or divorce.
+
+Two considerations are important:
+
+* Whether the event occurred within the frame of the given data
+* How much time it took until the event occurred
+
+In summary, this sub-chapter explains how to conduct survival analysis in mlr3.
+
+##Spatial Analysis
+
+[Spatial analysis](#spatial) data observations entail reference information about spatial characteristics.
+One of the largest shortcomings of spatial data analysis is the inevitable auto-correlation in spatial data.
+Auto-correlation is especially severe in data with marginal spatial variation.
+
+This sub-chapter provides instructions on how to handle the problems associated with spatial data accordingly.
+
+##Ordinal Analysis
+
+Coming soon!
+
+##Functional Analysis
+
+[Functional anlaysis](#functional) contains data that consists of curves varying over a continuum e.g. time, frequency or wavelength.
+This type of analysis is frequently used when examining measurements over a period of time.
+
+Steps on how to accommodate functional data structures in mlr3 are explained in the functional analysis-chapter.
+
+##Multilabel Classification
+
+[Multilabel analysis](#multilabel) deals with objects that can belong to more than one category at the same time.
+Numerous target labels are attributed to a single observation.
+Working with multilabel data requires one to use modified algorithms, to accommodate data specific characteristics.
+Two approaches to multilabel classification exist, namely the problem transformation method and the algorithm adaption method.
+
+Instructions on how to deal with multilabel analysis in mlr3 can be found in this sub-chapter.
+
+##Cost Sensitive Classification
```

Comment:
Put a whitespace between the "##" and the header names

Applies to all headers above